src/Codeception/Module/WebDriver.php

```diff
@@ -511,11 +507,13 @@ public function _getCurrentUri()
 
     public function _saveScreenshot($filename)
     {
-        if ($this->webDriver !== null) {
+        try {
+            if ($this->webDriver === null) {
+                throw new \Exception("WebDriver::_saveScreenshot method has been called when webDriver is not set");
```

Comment:
I am not sure if it is a very good idea to throw exception and catch it immediately,
wouldn't it be better to print some debug output and return?